src/common/components/CustomIcon/index.ts

```diff
@@ -9,3 +9,4 @@ export * from "./MinusIcon";
 export * from "./StarLineAltIcon";
 export * from "./WarningCircleIcon";
 export * from "./XCloseIcon";
+export * from "./SchoolIcon/BaseIcon";
```

Comment:
Can we rename "BaseIcon" to "SchoolIcon" since the function it exports is SchoolIcon?

Reply:
Since index already exports BaseIcon, I think we can leave it as 'export * from "./SchoolIcon"'. I think it should work!